prow/bump.sh

```diff
@@ -30,17 +30,6 @@ if ! (${SED} --version 2>&1 | grep -q GNU); then
   exit 1
```

Comment:
Is bump.sh still being used somewhere? It looks like this prow instance is bumped using the generic-autobumper instead: https://github.com/GoogleCloudPlatform/oss-test-infra/blob/0412bc0d9d15f2e1d1f27bbbb256950d4f0a0d07/prow/prowjobs/GoogleCloudPlatform/oss-test-infra/gcp-oss-test-infra-config.yaml#L81-L153

I suspect this script and the reference to it in the README.md are outdated.

Reply:
I think it's only for manual updates. It was working before I made the change, so I think fixing it is legit enough, but I'm also okay to just drop it from the repo. (It's referencing a few very specific files anyways and doesn't cover all of what the autobump does).

Reply:
/shrug
SGTM
/meowvie tools